src/features/HistorySideBar.test.tsx

```diff
@@ -5,6 +5,12 @@ import { EVENT_NAMES_TO_CHAT } from "../events";
 
 import { ChatHistoryItem } from "../hooks/useChatHistory";
 
+vi.mock("../hooks/useEventBusForChat", () => ({
```

Comment:
I don't think  useEventBusrForChat needs to be mocked, it seems that the last call to the spy is from request caps.

It's a small update to the existing test, mock uuid and change the expected last call to `postMessageSpy`
```tsx
  it("start new chat", async () => {
    vi.mock("uuid", () => ({ v4: () => "foo" }));
    const { user, ...app } = render(<HistorySideBar />);

    const postMessageSpy = vi.spyOn(window, "postMessage");

    const startNewChatButton = app.getByText("Start a new chat");

    await user.click(startNewChatButton);

    expect(postMessageSpy).toHaveBeenCalledWith({type: EVENT_NAMES_TO_CHAT.NEW_CHAT }, "*");

    expect(postMessageSpy).toHaveBeenLastCalledWith({ type: EVENT_NAMES_FROM_CHAT.REQUEST_CAPS, payload: { id: "foo" } }, "*")

});


```